integration-tests/testsuite/src/test/java/io/apicurio/tests/dbupgrade/KafkaSqlLogCompactionIT.java

```diff
@@ -0,0 +1,152 @@
+import io.apicurio.tests.common.interfaces.TestSeparator;
+import io.apicurio.tests.common.utils.RegistryUtils;
+
+/**
+ * @author Fabian Martinez
+ */
+@DisplayNameGeneration(SimpleDisplayName.class)
+@TestInstance(Lifecycle.PER_CLASS)
+@Tag(Constants.DB_UPGRADE)
+public class KafkaSqlLogCompactionIT implements TestSeparator, Constants {
+
+    final Logger logger = LoggerFactory.getLogger(getClass());
+
+
+    @Test
+    public void testLogCompaction() throws Exception {
+
+        RegistryStorageType previousStorageValue = RegistryUtils.REGISTRY_STORAGE;
+
+        String testName = "testLogCompaction";
+        RegistryStorageType storage = RegistryStorageType.kafkasql;
+
+        RegistryUtils.REGISTRY_STORAGE = storage;
+
+        Path logsPath = RegistryUtils.getLogsPath(getClass(), testName);
+        RegistryFacade facade = RegistryFacade.getInstance();
+
+        try {
+
+            Map<String, String> appEnv = facade.initRegistryAppEnv();
+
+            //runs all required infra except for the registry
+            facade.deployStorage(appEnv, storage);
+
+            //Create the kafkasql-journal topic with aggressive log compaction
+            //bin/kafka-topics.sh --bootstrap-server localhost:9092 --create --topic kafkasql-journal --partitions 1 --replication-factor 1 --config min.cleanable.dirty.ratio=0.000001 --config cleanup.policy=compact --config segment.ms=100 --config delete.retention.ms=100
+            var kafkaAdminClient = KafkaFacade.getInstance().adminClient();
+            var journal = new NewTopic("kafkasql-journal", 1, (short)1);
+            journal.configs(Map.of(
+                        "min.cleanable.dirty.ratio","0.000001",
+                        "cleanup.policy","compact",
+                        "segment.ms", "100",
+                        "delete.retention.ms", "100"
+                    ));
+            kafkaAdminClient.createTopics(Arrays.asList(journal));
+
+            appEnv.put("QUARKUS_HTTP_PORT", "8081");
+
+            String registryNameSuffix = "first-start";
+            facade.runRegistry(appEnv, registryNameSuffix, "8081");
+            facade.waitForRegistryReady();
+            //
+
+            var registryClient = RegistryClientFactory.create("http://localhost:8081");
+
+            createArtifact(registryClient, ArtifactType.AVRO, ApicurioV2BaseIT.resourceToString("artifactTypes/" + "avro/multi-field_v1.json"));
+            var artifactdata = createArtifact(registryClient, ArtifactType.JSON, ApicurioV2BaseIT.resourceToString("artifactTypes/" + "jsonSchema/person_v1.json"));
+            createArtifact(registryClient, ArtifactType.PROTOBUF, ApicurioV2BaseIT.resourceToString("artifactTypes/" + "protobuf/tutorial_v1.proto"));
+
+            assertEquals(3, registryClient.listArtifactsInGroup(null).getCount());
+
+            //spend some time doing something
+            //this is just to give kafka some time to be 100% the topic is log compactedsddddddd
```

Comment:
What happened at the end of this line?  :)

Reply:
hahaha it's funny